src/modules/addScore.js

```diff
@@ -0,0 +1,18 @@
+async function addScore(user, score, URL) {
+  await fetch(URL, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({ user: user.value, score: score.value }),
+  })
+    .then((res) => res.json())
+    .then((data) => {
+      console.log(data);
+    });
+
+  user.value = '';
+  score.value = '';
+}
+
+export default addScore;
```

Comment:
- [ ] Kindly use `arrow functions` instead of the `function` keyword for the `addScore` function as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Use%20arrow%20functions%20instead%20of%20the%20function%20keyword.).